### DISCUSSION TITLE
Import Organization Line Breaks

### DISCUSSION THREAD

**Discussion by shellscape** [biomejs/biome#661] [Q&A] [answered]
Hey folks. Investigating Biome to replace ESLint and Prettier in a few projects. Is there a configuration or option for inserting line breaks between import groups when formatting? Couldn't spot that in the docs and tried a few search terms in the repo that didn't yield much.

**Comment by ematipico** [comment, marked as answer]
The isn't at the moment, although there's a discussion about it

https://github.com/biomejs/biome/discussions/645
